cypress/integration/Case07_Send_money.js

```diff
@@ -5,123 +5,136 @@ import HomePage from '../PageObjects/HomePage'
-  //       cy.log(sendMoneyUrl)
-  //       const moneyLink = sendMoneyUrl
-  //       const pattern = /(?:http[s]?:\/\/)[^\s[",><]*/gim
-  //       const validMoneyLnk = moneyLink.match(pattern)
-  //       cy.log(validMoneyLnk)
-  //       SendMoneyPage.doneButton.click()
-  //       cy.clearLocalStorage()
-  //       cy.clearCookies()
+  //   SendMoneyPage.nameInput.type('Name Surname')
+  //   SendMoneyPage.nextButton.click()
+  //   SendMoneyPage.moneyInput.type('0.01')
+  //   SendMoneyPage.nextButton.click()
+  //   SendMoneyPage.messageInput.type('test money')
+  //   SendMoneyPage.nextButton.click()
+  //   SendMoneyPage.confirmButton.click()
+  //   SendMoneyPage.copyLinkButton.click()
+  //   SendMoneyPage.doneButton.should('be.visible')
+
+  //   //get link from clipboard
+  //   //cy.task('getClipboard').then(sendMoneyUrl => {
+
+  //     cy.get('[data-testid*="http"]').invoke('attr', 'data-testid').then(sendMoneyUrl => {
+  //     cy.log(sendMoneyUrl)
+  //     const moneyLink = sendMoneyUrl
+  //     const pattern = /(?:http[s]?:\/\/)[^\s[",><]*/gim
+  //     const validMoneyLnk = moneyLink.match(pattern)
+  //     cy.log(validMoneyLnk)
+  //     SendMoneyPage.doneButton.click()
+  //     cy.clearLocalStorage()
+  //     cy.clearCookies()
+  //     cy.readFile('cypress/fixtures/userMnemonicSave.txt').then(mnemonic => {
   //       StartPage.open()
   //       StartPage.signInButton.click()
   //       LoginPage.recoverFromPassPhraseLink.click()
   //       LoginPage.pageHeader.should('contain', 'Recover')
-  //       LoginPage.mnemonicsInput.type(Cypress.env('additionalAccountMnemonics'))
+  //       LoginPage.mnemonicsInput.type(mnemonic)
   //       LoginPage.recoverWalletButton.click()
   //       LoginPage.yayButton.click()
+
   //       HomePage.claimButton.should('be.visible')
   //       HomePage.moneyAmountDiv.invoke('text').then(moneyBefore => {
   //         cy.log('Money before sending: ' + moneyBefore)
   //         cy.visit(validMoneyLnk.toString())
 
-  //         //wait for blockchain payment
   //         cy.contains('Claim').should('be.visible')
   //         HomePage.moneyAmountDiv.invoke('text').should('eq', (Number(moneyBefore) + 0.01).toFixed(2))
   //         SendMoneyPage.yayButton.click()
   //       })
   //     })
   //   })
   // })
+
+  // test send with verification
+  it('User is able to send money', () => {
+    localStorage.clear()
+    cy.readFile('../GoodDAPP/cypress/fixtures/userMnemonicSave.txt').then(mnemonic => {
+      StartPage.open()
+      StartPage.signInButton.click()
+      LoginPage.recoverFromPassPhraseLink.click()
+      LoginPage.pageHeader.should('contain', 'Recover')
+      LoginPage.mnemonicsInput.type(mnemonic)
+      LoginPage.recoverWalletButton.click()
+      LoginPage.yayButton.click()
+
+      HomePage.claimButton.click()
+
+      cy.request('POST', 'https://good-server.herokuapp.com/admin/queue', [ { password: 'password' }, { allow: 1 }, ])
```

Comment:
creds shouldn't be hard-coded. 

Please, **use env value** for gundb password:

- locally you could test it via .env.test but please **don't commit it**
- ask @sirpy to configure it on CI server. 

P.S.
from my point of view, regarding we have to set some admin creds just for test, the better option would be to use /user/enqueue mocking. But we're not having time for it, _let do it after_, once tests will pass on each env. Now, just add env var for gundbpassword and configure it on CI server

Reply:
Also, you're sending the request despite is user awainit on queue or not. Please add some simple UI checks for it, **do not send request if it's not needed**